package.json

```diff
@@ -90,5 +90,8 @@
     "packages/tools",
     "packages/playground",
     "packages/create-package"
-  ]
+  ],
```

Comment:
Please remove this, the monorepo should not have this dependency

Reply:
Done.